app-modules/engagement/src/Notifications/EngagementBatchFinishedNotification.php

```diff
@@ -70,7 +77,7 @@ public function toEmail(object $notifiable): MailMessage
         }
 
         return $message
-            ->subject('Bulk Engagements Finished Processing Successfully.')
+            ->subject($this->title)
```

Comment:
Changing the email subject is not mentioned in the scope of the ticket. Please check with Product in the ticket to explain what the subject is now, and whether or not it should also be changed as a part of this ticket? And if so, to what?